### PR TITLE
Gravity generator glows.

### LINKED ISSUE #3583: Gravity Generator Should Emit Light
The gravity generator should probably emit light when on (and maybe a little while charging).
Looks a bit strange with the lights off:

Here's `LIGHT_COLOR_CYAN` with range `4`:

### PR DESCRIPTION
**Pull request by chaoko99**
Resolves #3583

### CODE REVIEW

**Comment by chaoko99** [comment]
resolved.

**Comment by Lohikar** [comment]
Will test when I have a moment.

File: code/modules/power/gravitygenerator.dm [hunk 1]
```diff
@@ -88,13 +88,17 @@
...
+		set_light(8,1,LIGHT_COLOR_CYAN)
+
 //
 // Generator an admin can spawn
 //
 
 /obj/machinery/gravity_generator/main/station/admin/Initialize()
 	. = ..()
 	round_start = 1
+	set_light(8,1,LIGHT_COLOR_CYAN)
```
Comment (Lohikar) [inline code/modules/power/gravitygenerator.dm]: Would like this converted to just setting the vars on the type.
Also, why only the `admin` type (whatever that is)?

Comment (chaoko99) [reply code/modules/power/gravitygenerator.dm]: fiiixed?

**Comment by skull132** [comment]
:ree: post a picture next time.

**Pull request merged**